open-machine-learning-jupyter-book/assignments/ml-fundamentals/Build-classification-models-Predict-the-price-range.ipynb

```diff
@@ -0,0 +1,1096 @@
+{
```

Comment:
<p>a space before <a href="https://github.com/Nyandwi/machine_learning_complete" target="_blank">machine_learning_complete</a>. A dot at the end.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='96'/>